draft-ietf-core-dns-over-coap.md

```diff
@@ -511,6 +512,18 @@ harden against injecting spoofed responses.
 Consequently, it is of little concern to leverage the benefits of CoAP caching by setting the ID to
 0.
 
+A user of DoC must be aware that the DoC server
+may communicate unencrypted with the upstream DNS infrastructure, e.g., using DNS over UDP.
+DoC can only guarantee confidential communication and integrity between parties for which the
+security context is exchanged.
+The DoC server may use another security context to communicate confidentially and with integrity
+upstream (e.g., DNS over QUIC {{-doq}}) or just integrity (e.g., DNSSEC {{-dnssec}}), but, while
+recommended, this is opaque to the DoC client on the protocol level.
+
+If the network or node constraints do not allow for the DoC client to receive DNSSEC information via
+the DoC server, it might want to establish a trust relation at least with the DoC server in other ways.
+How this trust relation is established is out of scope of this document.
```

Comment:
```suggestion
A DoC client may not be able to perform DNSSEC validation,
eg. due to code size constraints, or due to size of the responses.
It MAY trust its DoC server to perform DNSSEC validation;
how that trust is expressed is out of scope of this document.
For example, if the DoC client is configured to use a particular credential by which to recongize an eligible DoC server,
information about whether or not that server can be trusted to have performed DNSSEC validation can be configured along with it.
```

Reply:
Thanks. I tweaked the last sentence for readability though:

```suggestion
A DoC client may not be able to perform DNSSEC validation,
e.g., due to code size constraints, or due to size of the responses.
It may trust its DoC server to perform DNSSEC validation;
how that trust is expressed is out of scope of this document.
A DoC client may be, for instance, configured to use a particular credential by which it recognizes an eligible DoC server.
That information can also imply trust in the DNSSEC validation by that server.
```